public/models/interfaces.ts

```diff
@@ -102,6 +102,8 @@ export type Detector = {
   enabled?: boolean;
   enabledTime?: number;
   disabledTime?: number;
+  curState: DETECTOR_STATE;
+  initializationError: string;
```

Comment:
ha, I guess those are for `Detector state page`. but not a blocker for approving it.

Reply:
Yeah, correct